crates/typst-ide/src/lib.rs

```diff
@@ -92,7 +96,7 @@ fn summarize_font_family<'a>(variants: impl Iterator<Item = &'a FontInfo>) -> Ec
 }
 
 #[cfg(test)]
-mod tests {
+pub mod tests {
```

Comment:
why is this made public?

Reply:
This is related to the [discord message](https://discord.com/channels/1054443721975922748/1229699205786898485/1248923723298242560). But yeah, I should open another PR if I want to use `TestWorld` outside of crate.

> For typst-ide, internal functions just prevent developers from constructing their code analysis based on existing code of typst-ide. In typst crate it looks sensible to prefer private, but typst-ide we could prefer pub.